aisynphys/database/schema/dataset.py

```diff
@@ -140,11 +142,17 @@ def stimulus_tseries(self):
         # ('first_spike', 'stim_spike.id', 'The ID of the first spike evoked by this pulse'),
         ('data', 'array', 'Numpy array of presynaptic recording sampled at '+sample_rate_str, {'deferred': True}),
         ('data_start_time', 'float', "Starting time of the data chunk, relative to the beginning of the recording"),
+        
+        ('position', 'object', '3D location of this stimulation in the arbitrary coordinate system of the experiment'),
+        ('device_name', 'str', 'The name of the device used for this stimulus'),
```

Comment:
Is this redundant with `stim_pulse.recording.device_name`?

Reply:
Yes. fixed.